lib/api/src/native.rs

```diff
@@ -109,6 +109,23 @@ macro_rules! impl_native_traits {
         {
             /// Call the typed func and return results.
             pub fn call(&self, $( $x: $x, )* ) -> Result<Rets, RuntimeError> {
+                self.do_call(false, $( $x, )* )
+            }
+
+            /// Call the typed func and return results (unchecked)
```

Comment:
Same comment here, what's the difference between `call` and `call_unchecked`? Maybe we should explain this concept in the `Function` module once, and refers to this explanation everytime we need to (like here).